usaspending_api/submissions/helpers.py

```diff
@@ -198,3 +198,55 @@ def get_latest_submission_ids_for_fiscal_year(fiscal_year: int):
         .values_list("submission_id", flat=True)
     )
     return submission_ids
+
+
+def _get_latest_submission_ids_for_each_fiscal_quarter(
+    federal_account_id_filter_obj, fiscal_years: List[int], federal_account_id: int
+):
+    filters = {"submission_window__submission_reveal_date__lte": now()}
+    if len(fiscal_years) > 0:
+        filters["reporting_fiscal_year__in"] = fiscal_years
+    if federal_account_id:
+        filters[f"{federal_account_id_filter_obj}__treasury_account__federal_account_id"] = federal_account_id
+
+    cte = With(
```

Comment:
How does this work if you are selecting multiple Fiscal Years? The `latest_fiscal_period` is different for 2021 and 2020.

Reply:
This results in a `GROUP BY` on `toptier_code`, `reporting_fiscal_year`, and `reporting_fiscal_quarter`, so it's getting the max `reporting_fiscal_period` for each Quarter of each Fiscal Year